docs/cn/index.md

```diff
@@ -1,4 +1,4 @@
----
+﻿---
```

Comment:
restore this change?

Reply:
How to do that？

Reply:
This change is unnecessary. If you look at the button of 'show rich diff', it broke the table display.

Reply:
OK. Now it's fixed.